docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -91,17 +90,17 @@ All the published events of this type are then forwarded to an HTTP endpoint cal
   Kyma Dashboard
   </summary>
 
-1. In your Function's view, go to **Configuration** and click **Create Subscription+**.
-2. Provide the following parameters:
+1. Go to **Namespaces** and select the default Namespace.
+2. Go to **Configuration** > **Subscriptions** and click **Create Subscription+**.
+3. Provide the following parameters:
```

Comment:
(nit) 
maybe it's better to set the fields in the same sequence they are in the view itself ?
<img width="1382" alt="image" src="https://user-images.githubusercontent.com/15933100/211812713-95dc8497-2b08-4a35-9470-879c12f35d49.png">

Reply:
updated at all places.